apps/communications/contacts/js/views/details.js

```diff
@@ -82,6 +83,8 @@ contacts.Details = (function() {
 
     initPullEffect(cover);
 
+    LazyLoader.load('/contacts/js/nfc.js');
```

Comment:
I don't think that it should be on `contacts.js`, as only details use it (plus, `initDetails` is called from different places, and we should load nfc before all of them). 
I agree on the race conditions problem, but I think it's still better to just add the listeners when the load is done.
```js
LazyLoader.load('/contacts/js/nfc.js', ( ) => {
  utils.listeners.add({
      '#toggle-favorite': toggleFavorite,
      '#details-view-header': [
        {
          event: 'action',
          handler: handleDetailsBack
        }
      ],
      '#edit-contact-button': showEditContact
    });
});
```
...or any other proposal that doesn't use contacts.js for the load.